Userland/Applications/Piano/AudioPlayerLoop.cpp

```diff
@@ -139,10 +140,12 @@ void AudioPlayerLoop::write_wav_if_needed()
             m_track_manager.reset();
             m_track_manager.set_should_loop(false);
             do {
+                // FIXME: This progress detection is crude, but it works for now.
+                m_wav_percent_written.store(static_cast<int>(static_cast<float>(m_track_manager.transport()->time()) / roll_length * 100.0f));
                 m_track_manager.fill_buffer(m_buffer);
                 wav_writer.write_samples(m_buffer.span());
             } while (m_track_manager.transport()->time());
-            // FIXME: Make sure that the new TrackManager APIs aren't as bad.
```

Comment:
This `FIXME` is removed, was it solved already? What was the reason?

Reply:
Rebase error